stdlib/public/core/StaticString.swift

```diff
@@ -19,11 +19,55 @@
 
 /// A string type designed to represent text that is known at compile time.
 ///
-/// Instances of the `StaticString` type are immutable. `StaticString` provides
-/// limited, pointer-based access to its contents, unlike Swift's more
-/// commonly used `String` type. A static string can store its value as a
-/// pointer to an ASCII code unit sequence, as a pointer to a UTF-8 code unit
-/// sequence, or as a single Unicode scalar value.
+/// Instances of the `StaticString` type are immutable.
+///
+/// `StaticString` provides only low-level access to its contents, unlike
+/// Swift's more commonly used `String` type.
+///
+/// A static string can store:
```

Comment:
Close up these lines:
```suggestion
/// `StaticString` provides only low-level access to its contents, unlike
/// Swift's more commonly used `String` type. A static string can use
/// either of the following as its storage:
```